cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Person.java

```diff
@@ -0,0 +1,209 @@
+package no.unit.nva.cristin.person.model.nva;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+import static nva.commons.core.attempt.Try.attempt;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.net.URI;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import no.unit.nva.cristin.person.PersonUriUtils;
+import no.unit.nva.cristin.person.model.cristin.CristinPerson;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@JsonPropertyOrder({"@context"})
+public class Person {
+
+    @JsonInclude(NON_NULL)
+    @JsonProperty("@context")
+    private String context;
+    private final URI id;
+    private final List<NvaIdentifier> identifiers;
+    private final List<NvaIdentifier> names;
+    private final ContactDetails contactDetails;
+    private final URI image;
+    private final List<Affiliation> affiliations;
+
+    /**
+     * Creates a Person for serialization to client.
+     *
+     * @param context        The ontology context.
+     * @param id             Identifier of Person.
+     * @param identifiers    Different identifiers related to this object.
+     * @param names          Different names for this Person.
+     * @param contactDetails How to contact this Person.
+     * @param image          URI to picture of this Person.
+     * @param affiliations   This person's organization affiliations.
+     */
+    @JsonCreator
+    public Person(@JsonProperty("@context") String context, @JsonProperty("id") URI id,
```

Comment:
But that will change context for all instances of this class? What if we want the context field to be on the SearchResponse for a list of Person rather than on the Person itself?